nimble/host/audio/include/audio/ble_audio.h

```diff
@@ -562,6 +583,66 @@ struct ble_audio_event_codec_unregistered {
     const struct ble_audio_codec_record *record;
 };
 
+/** @brief BASS Source Removed */
+struct ble_audio_event_bass_remote_scan {
+    /** Connection Handle of Broadcast Assistant that is performing Scan procedure for us */
+    uint16_t conn_handle;
+};
+
+/** @brief BASS Source Added */
+struct ble_audio_event_bass_source_added {
+    /** Source ID */
+    uint8_t source_id;
+
+    /** Event status. 0 on success, BLE_HS Error code otherwise */
+    uint8_t status;
+};
+
+/** @brief BASS Source Modify */
+struct ble_audio_event_bass_source_modify {
+    /** Source ID */
+    uint8_t source_id;
+
+    /** Event status. 0 on success, BLE_HS Error code otherwise */
+    uint8_t status;
+
+    /** PA Sync operation to be performed by Broadcast Sink */
+    enum {
+        /** Do not synchronise to PA */
+        BLE_AUDIO_EVENT_BASS_NO_SYNC,
+
+        /** Synchronise to PA without using PAST */
+        BLE_AUDIO_EVENT_BASS_SYNC,
+
+        /** Terminate PA Sync */
+        BLE_AUDIO_EVENT_BASS_SYNC_TERMINATE,
+
+        /** Synchronise to PA using PAST */
+        BLE_AUDIO_EVENT_BASS_SYNC_PAST,
+    } pa_sync_operation;
```

Comment:
I'm not convinced it's needed, as the same data come in BASS Action event

Reply:
This will be left, but removed from audio event